attestation.md

```diff
@@ -91,9 +95,19 @@ attestation.store(attester).then(data => {
 
 ## Run 
 
-Execute the file by running this command in your terminal (still within your `kilt-rocks` directory):
+Execute the file by running this command in your terminal, still within your `kilt-rocks` directory:
+
 ```bash
 node 3-attestation.js
-``` 
-The `attestedClaim` should be printed out.   
-We now need to "send it back" to the claimer, for example by pasting it in the [shared doc](https://hackmd.io/c6OBNgWWR8yWJhMj7WICUA?edit).
+```
+
+You should see in your logs:
+* `true` for data validity;
+* The blockchain transaction status (TxStatus), which should be `Finalized`;
+* The `AttestedClaim` object.
+
+Copy the `AttestedClaim object`, you'll need it soon.
+
+Your job as an <span class="label-role attested">attester</span> is done: you've successfully attested a claim, written the attestation hash onto the chain, and prepared the `AttestedClaim` object for the claimer.
```

Comment:
Claimer and Claim are not highlighted as they are in other places.

e.g. `<span class="label-role claimer">claimer</span>`

Reply:
Fixed, ty.